tui/mainModel.go

```diff
@@ -335,47 +340,22 @@ func (m MainModel) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 				}
 
 			} else if m.activeTab == CONTAINERS {
+				// handles an action as background non-blocking task
 				switch {
 				case key.Matches(msg, ContainerKeymap.ToggleListAll):
 					m.dockerClient.ToggleContainerListAll()
 
 				case key.Matches(msg, ContainerKeymap.ToggleStartStop):
 					log.Println("s pressed")
-					curItem := m.getSelectedItem()
-					if curItem != nil {
-						containerId := curItem.(dockerRes).getId()
-						err := m.dockerClient.ToggleStartStopContainer(containerId)
+					go m.handleBackground(m.dockerClient.ToggleStartStopContainer)
```

Comment:
and update these to get current Id and abide by the newly suggested function signature for `m.handleBackground`.